src/devlog/utils/text.py

```diff
@@ -1,11 +1,26 @@
 import io
+import os
 import re
+import xml.etree.ElementTree as etree  # noqa: DUO107,N813
```

Comment:
Codacy found an issue: [Using xml.etree.ElementTree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace xml.etree.ElementTree with the equivalent defusedxml package, or make sure defusedxml.defuse_stdlib() is called.](https://app.codacy.com/manual/jarek.zgoda/devlog/pullRequest?prid=5774733)